components/expenses/Expense.js

```diff
@@ -500,8 +505,50 @@ class Expense extends React.Component {
           {this.state.error && (
             <MessageBox type="error" withIcon mb={2}>
               {formatErrorMessage(intl, this.state.error)}
+              <br />
+              {formatErrorMessage(intl, this.state.error).startsWith(
+                'Host has two-factor authentication enabled for large payouts.',
+              ) && (
+                <FormattedMessage
+                  id="PayExpenseModal.HostTwoFactorAuthEnabled"
+                  defaultMessage="Please go to your {settings} to enable two-factor authentication on your account."
+                  values={{
+                    settings: (
+                      <StyledLink as={Link} route="editCollective" params={{ slug: LoggedInUser.collective.slug }}>
+                        account settings
+                      </StyledLink>
+                    ),
+                  }}
+                />
```

Comment:
Here `account settings` won't be translated (ie. in French you'll get something like `Rendez vous dans vos account settings pour activer (...)`). I suggest using the approach documented in https://docs.opencollective.com/help/contributing/development/translations:

```suggestion
                <FormattedMessage
                  id="PayExpenseModal.HostTwoFactorAuthEnabled"
                  defaultMessage="Please go to your <SettingsLink>account settings</SettingsLink> to enable two-factor authentication on your account."
                  values={{
                    SettingsLink: getI18nLink({ 
                      as: Link, 
                      route: 'editCollective', 
                      params: { slug: LoggedInUser.collective.slug } 
                    }),
                  }}
                />
```